public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -502,7 +504,7 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
   }
 
   public duplicatePanel(vizPanel: VizPanel) {
-    this.state.body.duplicatePanel(vizPanel);
+    getLayoutManagerFor(vizPanel).duplicatePanel(vizPanel);
```

Comment:
I think this approach can also be used for `removePanel`, right?

Reply:
yes, updated that as well